### ISSUE TITLE
[Accessibility] Color Contrast is not respected.

### ISSUE THREAD

**Issue by andreacantone** [iSoron/uhabits#666]
## Description

The contrast ratio between text and background should be at least 4.5 . Some elements don't respect the contrast ratio. 

## How to reproduce the error

1. Activate the app [Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=it&gl=US)
2. Open the app [Loop Habit Tracker](https://play.google.com/store/apps/details?id=org.isoron.uhabits&hl=it&gl=US)
3. Activate the registration from the Accessibility Scanner and use the Loop Habit Tracker app.
4. After using the app, stop the recording.

## Screenshot

The results will show the errors found in the app. For example:

![IMG_20201118_172712](https://user-images.githubusercontent.com/25429624/99558873-75b42080-29c4-11eb-83d8-92b11ffb2720.png)

**Comment by iSoron** [comment]
Thank you for pointing this out, @andreacantone, but this is restricted to a single line of text and the contrast ratio is not much lower than the suggestion, so I am not planning to correct this.

**Comment by localpcguy** [comment]
This is significantly poor for the dark mode version. Is this something you'd consider reopening and maybe a PR to address?

**Comment by hiqua** [comment]
What's the ratio for the dark mode? PRs are always welcome, no guarantee it'll be merged but it's worth a try!
